package.json

```diff
@@ -11,10 +11,25 @@
     "test:cc": "yarn ng test -- --watch=false --cc",
     "test:xvfb": "docker run --rm -v `pwd`:/ipaas-ui:z -w /ipaas-ui -u `id -u` docker.io/rhipaas/karma-xvfb ./karma-xvfb.sh",
     "pree2e": "webdriver-manager update",
-    "e2e": "protractor",
+    "e2e": "better-npm-run e2e:local",
+    "e2e:ipaas-qe": "better-npm-run e2e:ipaas-qe",
     "sw": "sw-precache --root=dist --config=sw-precache-config.js",
     "docker:build": "yarn ng build -- --aot --prod && docker build -t rhipaas/ipaas-ui:latest ."
   },
+  "betterScripts": {
+    "e2e:local": {
+      "command": "protractor",
+      "env": {
+        "TEST_ENV": "local"
+      }
```

Comment:
included new dependency `npm-better-run` because it allows us to specify shell env variables in package.json.
Again - expecting more customizations while we tune up CI/CD.